doc/classes/VisualServer.xml

```diff
@@ -1374,6 +1374,17 @@
 				Sets the flag for a given [enum InstanceFlags]. See [enum InstanceFlags] for more details.
 			</description>
 		</method>
+		<method name="instance_geometry_set_material_overlay">
+			<return type="void">
+			</return>
+			<argument index="0" name="instance" type="RID">
+			</argument>
+			<argument index="1" name="material" type="RID">
+			</argument>
```

Comment:
This should be updated to follow the new formatting (running `godot --doctool` would do it, otherwise applying the diff from https://github.com/godotengine/godot/runs/4218129894?check_suite_focus=true ).

Reply:
Amended